picard/i18n.py

```diff
@@ -93,13 +93,13 @@ def _ugettext_countries(msg):
         def _ugettext_attributes(msg):
             return msg
 
-    builtins.__dict__['ungettext'] = _ungettext
+    builtins.__dict__['lngettext'] = _lngettext
```

Comment:
This change requires few more changes:

https://github.com/metabrainz/picard/blob/master/setup.cfg#L8

In order to extract strings used as `lngettext()` arguments, you may need to modify https://github.com/metabrainz/picard/blob/master/setup.py#L423
because `lngettext` isn't part of default keywords (see https://github.com/python-babel/babel/blob/master/babel/messages/extract.py#L32)